packages/api-client-core/src/GadgetConnection.ts

```diff
@@ -328,20 +328,21 @@ export class GadgetConnection {
   }
 
   private newBaseClient() {
-    const exchanges = [dedupExchange];
+    const exchanges = [];
 
     // apply urql's default caching behaviour when client side (but skip it server side)
     if (typeof window != "undefined") {
       exchanges.push(cacheExchange);
     }
 
     exchanges.push(
-      multipartFetchExchange,
+      fetchExchange,
       subscriptionExchange({
-        forwardSubscription: (operation) => {
+        forwardSubscription: (request) => {
           return {
             subscribe: (sink) => {
-              const dispose = this.baseSubscriptionClient.subscribe(operation, sink as Sink<ExecutionResult>);
+              const input = { ...request, query: request.query || "" };
+              const dispose = this.baseSubscriptionClient.subscribe(input, sink as Sink<ExecutionResult>);
```

Comment:
This type changed between versions of urql -- the query is now options for some reason, so this maps to the kind of object that graphql-ws is expecting